drivers/video/mxsfb.c

```diff
@@ -34,6 +36,22 @@ __weak void mxsfb_system_setup(void)
 {
 }
 
+static int setup;
+static struct fb_videomode fbmode;
+static int depth;
+
+int mxs_lcd_panel_setup(struct fb_videomode mode, int bpp,
+	uint32_t base_addr)
```

Comment:
U-Boot patch checker: Alignment should match open parenthesis.
```suggestion
			uint32_t base_addr)
```